frontend/src/components/HospitalInventoryEdit.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from 'react'
+import Col from 'react-bootstrap/Col'
+import Form from 'react-bootstrap/Form'
+import Card from 'react-bootstrap/Card'
+import Button from 'react-bootstrap/Button'
+import { useParams } from 'react-router-dom'
+import style from './HospitalInventoryEdit.module.css'
+import { HospitalInventory, HospitalAddress, fetchHospitalInventory, fetchHospitalAddress } from '../apis/infomed'
+
+
+interface HospitalInventoryParams {
+  hospitalId: string
+}
+
+export const HospitalInventoryEdit = ({hospitalId}: HospitalInventoryParams) => {
+
+  const [inventoryList, setInventoryList] = useState<HospitalInventory[]>([]);
+  const [hospitalAddress, setHospitalAddress] = useState<HospitalAddress[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const list = await fetchHospitalInventory(hospitalId);
+      setInventoryList(list);
+      
+      const address = await fetchHospitalAddress(hospitalId);
+      setHospitalAddress(address);
```

Comment:
As I wrote above, the hospital address and inventory could probably be fetched all at once, but I'd defer this to our backend folks. However, if not, then I'd like to introduce you to `Promise.all`. Each time you `await` a particular promise, you end up wasting some time. If you wait for one promise, and then for another, it's only worth doing it if the result of the second promise actually depends on the result of the first one. However, in this case, both results only depend on `hospitalId`, and they can be fetched in parallel. I would do it like this:

```typescript
const [list, address] = await Promise.all([
  fetchHospitalInventory(hospitalId),
  fetchHospitalAddress(hospitalId),
]);
```